prov/gni/src/gnix_fabric.c

```diff
@@ -484,6 +484,9 @@ static int _gnix_ep_getinfo(enum fi_ep_type ep_type, uint32_t version,
 				goto err;
 			}
 			mode = hints->mode & ~GNIX_FAB_MODES_CLEAR;
+			if (version < FI_VERSION(1, 5)) {
```

Comment:
Technically, no. FI_VERSION returns a uint32_t that is could just as easily be checked this way. 

However, this should be checked using that macro in the event that it should ever change.